DependencyInjection/DoctrineExtension.php

```diff
@@ -91,6 +92,21 @@ public function load(array $configs, ContainerBuilder $container)
         $config        = $this->processConfiguration($configuration, $configs);
 
         if (! empty($config['dbal'])) {
+            // if no DB connection defined, prepend an empty one for the default
+            // connection name in order to make Symfony Config resolve the
+            // default values
+            if (empty($config['dbal']['connections'])) {
+                $config = $this->processConfiguration($configuration, array_merge([
+                    [
+                        'dbal' => [
+                            'connections' => [
+                                ($config['dbal']['default_connection'] ?? 'default') => [],
+                            ],
+                        ],
+                    ],
+                ], $configs));
```

Comment:
I don't like how verbose these are, let's do something like
```suggestion
                $configs[] = ['dbal' => ['connections' => [($config['dbal']['default_connection'] ?? 'default') => []]]];
                $config    = $this->processConfiguration($configuration, $configs);
```
instead

Reply:
done 👍